core/server/common/src/main/java/alluxio/cli/extensions/command/InstallCommand.java

```diff
@@ -65,6 +66,10 @@ public String getDescription() {
   public int run(CommandLine cl) {
     String uri = cl.getArgs()[0];
     String extensionsDir = Configuration.get(PropertyKey.EXTENSIONS_DIR);
+    File dir = new File(extensionsDir);
+    if (!dir.exists() && !dir.mkdirs()) {
```

Comment:
this creates the directory locally, but we might install on other hosts - should we also add a remote mkdir before rsyncing to other nodes?

Reply:
rsync will create parent dir